Src/libCZI/utilities.cpp

```diff
@@ -323,6 +323,122 @@ tString trimImpl(const tString& str, const tString& whitespace)
 #endif
 }
 
+/*static*/std::map<std::wstring, std::wstring> Utilities::TokenizeAzureUriString(const std::wstring& input)
```

Comment:
[bike-shedding] would be nice if function was split up into contextual functions, but there is no need to extend tokenizer and nothing I can point out that could be problematic functionally.